### PR TITLE
Add user list

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra/pull/1101
https://github.com/open-orchestra/open-orchestra-libs/pull/71
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/64
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2013
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/133

### CODE REVIEW

File: UserBundle/Repository/UserRepository.php [hunk 1]
```diff
@@ -63,4 +65,169 @@ public function hasElementWithRole(RoleInterface $role)
...
+        return $this->hydrateAggregateQuery($qa);
+    }
+
+    /**
+     * @param PaginateFinderConfiguration $configuration
+     * @param array                       $sitesId
+     *
+     * @return array
+     */
+    public function findForPaginateFilterBySitesId(PaginateFinderConfiguration $configuration, array $sitesId)
```
Comment (ngilain) [inline UserBundle/Repository/UserRepository.php]: @alavieille findForPaginateFilterBySiteIds

File: UserBundle/Repository/UserRepository.php [hunk 2]
```diff
@@ -63,4 +65,169 @@ public function hasElementWithRole(RoleInterface $role)
...
+
+        return $this->countDocumentAggregateQuery($qa);
+    }
+
+    /**
+     * @param array $sitesId
+     *
+     * @return int
+     */
+    public function countFilterBySitesId(array $sitesId)
```
Comment (ngilain) [inline UserBundle/Repository/UserRepository.php]: countFilterBySiteIds

File: UserBundle/Repository/UserRepository.php [hunk 3]
```diff
@@ -63,4 +65,169 @@ public function hasElementWithRole(RoleInterface $role)
...
+        return $this->countDocumentAggregateQuery($qa);
+    }
+
+    /**
+     * @param PaginateFinderConfiguration $configuration
+     * @param array                       $sitesId
+     *
+     * @return int
+     */
+    public function countWithFilterAndSitesId(PaginateFinderConfiguration $configuration, array $sitesId)
```
Comment (ngilain) [inline UserBundle/Repository/UserRepository.php]: countWithFilterAndSiteIds

**Comment by alavieille** [comment]
@ngilain pr updated

**Comment by itkg-nanne** [comment]
:+1:

**Pull request merged**